docs/Reference/APIs/PrivacyEnhancementsAPIs.md

```diff
@@ -0,0 +1,30 @@
+# Privacy Enhancements  APIs
+## APIs
+### `getContractPrivacyMetadata` 
+The api to query the Privacy Metadata for a contract account address
+#### Parameter
+* contract address
+
+#### Returns
+* `creationTxHash`: ACOTH
+* `privacyFlag`: unsigned integer with `1` for PP, `3` for PSV contracts (default value is `0`).
+
+#### Examples
```

Comment:
Use the `!!! example` [admonition panel.](https://github.com/PegaSysEng/doc.common/wiki/MkDocs-And-Custom-Markdown-Guide#admonition) for examples to be immediately visually recognised by the user.

Reply:
Updated the examples paragraph with your recommended formatting.